fix: Fix milvus vector_name_exists bug

HYP-hu opened the pull request:
# Description

Fix the MilvusStore.vector_name_exists method which previously only checked whether a collection exists, but not whether it contains any data.

When testing ChatDB with Milvus as the vector store, this caused a bug: even if the collection was empty, it was treated as valid, leading to skipped document ingestion and missing database summaries during initialization.

This PR adds a row_count check to ensure the collection not only exists, but also contains data before returning True.

# How Has This Been Tested?

Test the ChatDB function using  Mivus + DB-GPT

# Snapshots:
 


# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have already rebased the commits and make the commit message conform to the project standard.
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged and published in downstream modules

Aries-ckt reviewed (approved):
LGTM.thanks for your contribution.

HYP-hu commented:
Sorry about that! I’ve removed the unused import and fixed the formatting. The latest commit passes ruff check. Appreciate your time reviewing it again!
